imports/server/migrations/37-role-reorganization.ts

```diff
@@ -0,0 +1,45 @@
+import { Meteor } from 'meteor/meteor';
+import { Migrations } from 'meteor/percolate:migrations';
+import { GLOBAL_SCOPE } from '../../lib/is-admin';
+
+Migrations.add({
+  version: 37,
+  name: 'Reorganize roles and eliminate inactiveOperator',
+  up() {
+    // Casts necessary to account for schema changes
+
+    Meteor.users.find({ roles: 'inactiveOperator' } as any).forEach((user) => {
+      Meteor.users.update(user._id, {
+        $push: { roles: 'operator' },
+      } as any, {
+        validate: false,
+        filter: false,
+      } as any);
+      Meteor.users.update(user._id, {
+        $pull: { roles: 'inactiveOperator' },
```

Comment:
I take it you can't do both the `$push` and the `$pull` on the same target field in the same update?

Reply:
Yeah, Mongo seems to throw an error if you try to perform two operations on the same field. (I assume because the ordering would be underspecified, even though it's not in this specific case)